apps/api/v2/src/ee/calendars/services/calendars.service.ts

```diff
@@ -187,6 +205,27 @@ export class CalendarsService {
       userId,
       calendarType
     );
+
+    await this.deleteCalendarCache(userId);
+  }
+
+  async deleteCalendarCache(userId: number) {
```

Comment:
Could we move the newly added cache functions to `CalendarsCacheService` and then re-use the cache service? Would help making this class smaller and separate responsibilities.